src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +25,25 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    public List<LineResponse> findAll() {
+        return lineRepository.findAll().stream().map(line->LineResponse.of(line)).collect(Collectors.toList());
```

Comment:
중요한 부분은 아니지만 아래와 같이도 사용할 수 있겠네요~

```suggestion
        return lineRepository.findAll().stream().map(LineResponse::of).collect(Collectors.toList());
```